### DISCUSSION TITLE
Member의 google_id 컬럼을 social_id 컬럼으로 변경하는 것에 대하여

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/01-doo-re-back#36] [논의 사항]
추후 소셜 로그인 기능의 확장을 고려했을 때, 소셜 플랫폼에서 받아온 고유 아이디 값을 저장하는 컬럼을 social_id라고 하는 게 더 좋을 것 같다고 생각했습니다!
그리고 소셜 플랫폼 유형을 저장할 수 있도록 social_type이라는 컬럼을 추가하면 좋을 것 같아요.

**Comment by shkisme** [comment]
리소스가 크지 않으니, 저도 추후 카카오, 깃헙 등등의 소셜 로그인 지원까지 고려한다면, 추가하는게 좋아보여요!
확장성까지 고려하시는 부분 배워가겠습니다 👍

**Reply by 02ggang9** [reply]
저도 social_id가 나중을 생각하면 더 좋을 것 같네요~!

**Comment by lcqff** [comment]
찬성입니다~!

**Comment by JJimini** [comment]
굿굿

**Comment by amaran-th** [comment]
만장일치로 social_id로 변경하는 것으로 결정.
